pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -2593,13 +2593,7 @@ in
   };
 
   aider-nvim = super.aider-nvim.overrideAttrs {
-    patches = [ ./patches/aider.nvim/fix-paths.patch ];
-
-    postPatch = ''
-      substituteInPlace lua/aider.lua --replace-fail '@aider@' ${aider-chat}/bin/aider
-      substituteInPlace lua/helpers.lua --replace-fail '@aider@' ${aider-chat}/bin/aider
-    '';
-    nvimRequireCheck = "aider";
+    runtimeDeps = [ aider-chat ];
```

Comment:
@teto I believe this was the intended use of this, correct? It was a patch that replaced 'aider' with a placeholder of '@aider@' that we used to just patch in the nix binary path.

Reply:
yes exactly. It might still be early to apply the change as we dont enable runtimeDeps by default yet (sadly, we can turn it on once it's documented, else it will break some workflows).

Reply:
Hmm... alright. So I should just recreate the patch until we can use this feature.

Reply:
yes. I've had some trouble finding time to write the wrapper doc. I hope to do it next week but anyone feel free to beat me to it :)